pyccel/ast/omp.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+This file contain the OpenMP types.
+"""
+
+from .basic import Basic
+
+class OmpAnnotatedComment(Basic):
+
+    """Represents an OpenMP Annotated Comment in the code.
+
+    Parameters
+    ----------
+
+    txt: str
+        statement to print
+
+    combined: List (Optional)
+        constructs to be combined with the current construct
+
+    Examples
+    --------
+    >>> from pyccel.ast.omp import OmpAnnotatedComment
+    >>> OmpAnnotatedComment('parallel')
+    OmpAnnotatedComment(parallel)
+    """
+    _attribute_nodes = ()
+    _is_multiline = False
+
+    def __init__(self, txt, combined=None):
+        self._txt = txt
+        self._combined = combined
+        super().__init__()
+
+    @property
+    def is_multiline(self):
+        """Used to check if the construct needs brackets."""
+        return self._is_multiline
+
+    @property
+    def name(self):
+        """Name of the construct."""
+        return ''
```

Comment:
Is this code ever used? If the class needs to override this property then it is probably better to leave this method out then an error is raised if it has been forgotten

Reply:
Never mind, I see it is used for several classes from `OMP_Task_Construct` onwards